com.playeveryware.eos/Runtime/Core/Config/EOSConfig.cs

```diff
@@ -200,6 +212,7 @@ protected EOSConfig() : base("EpicOnlineServicesConfig.json")
             "Platform option flags",
             3)]
         [JsonConverter(typeof(ListOfStringsToAuthScopeFlags))]
+        [ParsesToAuthScopeFieldValidator]
         public AuthScopeFlags authScopeOptionsFlags;
```

Comment:
This should no longer need the field validator. None of the field members that are `enum` types should need them, since it is not possible to have an invalid value for them - unless the JSON itself is edited - and in that case the error should be generated from the JSON data layer.

Reply:
This is kept and the test was changed to test for invalid input instead